keyboards/str_studio/cadence_16/readme.md

```diff
@@ -0,0 +1,23 @@
+# Cadence_16
+
+![Cadence_16](https://strstudio.net/_next/image?url=%2Fcadence-1.png&w=1920&q=100)
```

Comment:
This image needs to be rehosted on Imgur. Doesn't look like this link works anyway.